draft-demarco-oauth-status-attestations.md

```diff
@@ -260,19 +260,41 @@ POST /status HTTP/1.1
 Host: issuer.example.org
 Content-Type: application/x-www-form-urlencoded
 
-credential_pop=$CredentialPoPJWT
+credential_pop = [$CredentialPoPJWT]
 ~~~
 
 To validate that the Wallet Instance is entitled to request its Status Attestation,
 the following requirements MUST be satisfied:
 
-- The Credential Issuer MUST verify the signature of the `credential_pop` object using
-the public key contained in the Digital Credential;
-- the Credential Issuer MUST verify that it is the legitimate Issuer.
+- The Credential Issuer MUST verify the signature of each `credential_pop` object using
+the public key contained in each Digital Credential;
+- The Credential Issuer MUST verify that it is the legitimate Issuer.
 
 The technical and details about the `credential_pop` object
 are defined in the next section.
 
+## Credential_pop
+
+Since the Wallet may request one or more Status Attestations, issued by the same Credential Issuer, the `credential_pop` object MUST be an array.
+
+For each array element, the JWT contained into `credential_pop` object, MUST include the parameters defined in the following table.
```

Comment:
as mentioned, the credential_pop is now supposed to be an array, not a json object